tools/pythonpkg/tests/fast/arrow/test_arrow_replacement_scan.py

```diff
@@ -3,19 +3,13 @@
 import os
 import pandas as pd
 
-try:
-    import pyarrow.parquet as pq
-    import pyarrow.dataset as ds
-
-    can_run = True
-except:
-    can_run = False
+pa = pytest.importorskip("pyarrow")
+pq = pytest.importorskip("pyarrow.parquet")
+ds = pytest.importorskip("pyarrow.dataset")
 
 
 class TestArrowReplacementScan(object):
```

Comment:
Could we also add test to trigger the exceptions added?
e.g., throw InvalidInputException("Expected a 'arrow_array_stream' PyCapsule, got: %s", string(capsule.name()));

Reply:
I'll add a test for that 👍